Flake8 errors

Current `backend/` folder needs some cleaning up. Here's the output from `flake8`:
```console
./app/server/app.py:18:1: E303 too many blank lines (3)
./app/server/controllers/post.py:60:80: E501 line too long (84 > 79 characters)
./app/server/controllers/auth.py:3:1: F401 'pydantic.BaseModel' imported but unused
./app/server/controllers/auth.py:4:1: F401 'typing.Optional' imported but unused
./app/server/controllers/auth.py:6:80: E501 line too long (90 > 79 characters)
./app/server/controllers/auth.py:6:80: E231 missing whitespace after ','
./app/server/controllers/auth.py:7:1: F401 'fastapi.Depends' imported but unused
./app/server/controllers/auth.py:7:1: F401 'fastapi.FastAPI' imported but unused
./app/server/controllers/auth.py:7:1: F401 'fastapi.status' imported but unused
./app/server/controllers/auth.py:8:33: E231 missing whitespace after ','
./app/server/controllers/auth.py:10:7: E225 missing whitespace around operator
./app/server/controllers/auth.py:11:1: E302 expected 2 blank lines, found 0
./app/server/controllers/auth.py:19:1: W293 blank line contains whitespace
./app/server/controllers/auth.py:22:1: W293 blank line contains whitespace
./app/server/controllers/auth.py:23:5: E265 block comment should start with '# '
./app/server/controllers/auth.py:26:18: E203 whitespace before ':'
./app/server/controllers/auth.py:26:80: E501 line too long (94 > 79 characters)
./app/server/controllers/auth.py:27:18: E203 whitespace before ':'
./app/server/controllers/auth.py:34:1: W293 blank line contains whitespace
./app/server/controllers/auth.py:37:80: E501 line too long (95 > 79 characters)
./app/server/controllers/auth.py:40:80: E501 line too long (80 > 79 characters)
./app/server/controllers/auth.py:41:9: F841 local variable 'e' is assigned to but never used
./app/server/controllers/auth.py:44:1: W293 blank line contains whitespace
./app/server/controllers/auth.py:45:5: E303 too many blank lines (2)
./app/server/controllers/auth.py:45:80: E501 line too long (84 > 79 characters)
./app/server/controllers/auth.py:46:51: W291 trailing whitespace
./app/server/controllers/auth.py:48:1: E305 expected 2 blank lines after class or function definition, found 1
./app/server/controllers/auth.py:48:29: W292 no newline at end of file
./app/server/controllers/user.py:3:1: F401 'passlib.context.CryptContext' imported but unused
./app/server/controllers/user.py:4:1: F401 'fastapi.Depends' imported but unused
./app/server/controllers/user.py:4:34: E231 missing whitespace after ','
./app/server/controllers/user.py:4:41: E231 missing whitespace after ','
./app/server/controllers/user.py:9:1: E302 expected 2 blank lines, found 1
./app/server/controllers/user.py:12:15: E203 whitespace before ':'
./app/server/controllers/user.py:29:48: E231 missing whitespace after ':'
./app/server/controllers/user.py:30:80: E501 line too long (92 > 79 characters)
./app/server/controllers/user.py:38:1: E302 expected 2 blank lines, found 1
./app/server/controllers/user.py:39:55: E231 missing whitespace after ':'
./app/server/controllers/user.py:40:80: E501 line too long (107 > 79 characters)
./app/server/controllers/user.py:41:80: E501 line too long (99 > 79 characters)
./app/server/controllers/user.py:46:1: E302 expected 2 blank lines, found 1
./app/server/controllers/user.py:57:52: E231 missing whitespace after ':'
./app/server/routes/posts.py:8:80: E501 line too long (80 > 79 characters)
./app/server/routes/post.py:20:1: E302 expected 2 blank lines, found 1
./app/server/routes/post.py:21:80: E501 line too long (103 > 79 characters)
./app/server/routes/post.py:28:80: E501 line too long (90 > 79 characters)
./app/server/routes/post.py:34:80: E501 line too long (133 > 79 characters)
./app/server/routes/post.py:46:80: E501 line too long (82 > 79 characters)
./app/server/routes/user.py:3:1: F401 'fastapi.security.OAuth2PasswordRequestForm' imported but unused
./app/server/routes/user.py:3:50: E231 missing whitespace after ','
./app/server/routes/user.py:9:17: W291 trailing whitespace
./app/server/routes/user.py:22:1: E302 expected 2 blank lines, found 1
./app/server/routes/user.py:22:80: E501 line too long (120 > 79 characters)
./app/server/routes/user.py:28:1: E302 expected 2 blank lines, found 1
./app/server/routes/user.py:28:80: E501 line too long (111 > 79 characters)
./app/server/routes/user.py:32:40: E231 missing whitespace after ','
./app/server/routes/user.py:34:1: E302 expected 2 blank lines, found 1
./app/server/routes/user.py:41:80: E501 line too long (119 > 79 characters)
./app/server/routes/user.py:46:1: E302 expected 2 blank lines, found 1
./app/server/routes/user.py:46:80: E501 line too long (82 > 79 characters)
./app/server/models/post.py:30:5: E303 too many blank lines (2)
./app/server/models/user.py:4:1: E302 expected 2 blank lines, found 1
./app/server/models/user.py:14:23: E203 whitespace before ':'
./app/server/models/user.py:22:1: E302 expected 2 blank lines, found 1
./app/server/models/user.py:29:24: E203 whitespace before ':'
./app/server/models/user.py:30:27: E203 whitespace before ':'
./app/server/models/user.py:33:1: E302 expected 2 blank lines, found 0
./app/server/models/user.py:40:5: E303 too many blank lines (2)
./app/server/models/user.py:43:23: E203 whitespace before ':'

```

--- a/backend/app/server/app.py
+++ b/backend/app/server/app.py
@@ -14,7 +14,6 @@ app.include_router(PostRouter, tags=["Post"], prefix="/post")
 app.include_router(PostsRouter, tags=["Posts"], prefix="/posts")
 
 
-
 @app.get("/", tags=["Root"])
 async def read_root():
     return {"message": "Welcome to this fantastic app!"}

--- a/backend/app/server/controllers/auth.py
+++ b/backend/app/server/controllers/auth.py
@@ -1,13 +1,13 @@
-from passlib.context import CryptContext
 import jwt
-from pydantic import BaseModel
-from typing import Optional
 from datetime import datetime, timedelta
-from fastapi.security import OAuth2PasswordBearer, HTTPAuthorizationCredentials,HTTPBearer
-from fastapi import Depends, FastAPI, HTTPException, status, Security
-from dotenv import dotenv_values,find_dotenv
+from passlib.context import CryptContext
+from fastapi.security import OAuth2PasswordBearer, HTTPAuthorizationCredentials, HTTPBearer
+from fastapi import HTTPException, Security
+from dotenv import dotenv_values, find_dotenv
 
-config=dotenv_values(find_dotenv())
+config = dotenv_values(find_dotenv())
+
+
 class AuthHandler():
 
     security = HTTPBearer()
@@ -16,33 +16,33 @@ class AuthHandler():
 
     def get_password_hash(self, password):
         return self.pwd_context.hash(password)
-        
+
     def verify_password(self, plain_password, hashed_password):
         return self.pwd_context.verify(plain_password, hashed_password)
-    
-    #create jwt access token
+
+    # create jwt access token
     def create_access_token(self, email):
         payload = {
-            'exp' : datetime.utcnow() + timedelta(int(config["ACCESS_TOKEN_EXPIRE_MINUTES"])),
-            'sub' : email
+            'exp': datetime.utcnow() + timedelta(int(config["ACCESS_TOKEN_EXPIRE_MINUTES"])),
+            'sub': email
         }
         return jwt.encode(
             payload,
             config["SECRET_KEY"],
             algorithm=config["ALGORITHM"]
         )
- 
+
     def decode_token(self, token):
         try:
             payload = jwt.decode(token, config["SECRET_KEY"], algorithms=[config["ALGORITHM"]])
             return payload["sub"]
         except jwt.ExpiredSignatureError:
             raise HTTPException(status_code=401, detail='Signature has expired')
-        except jwt.InvalidTokenError as e:
+        except jwt.InvalidTokenError:
             raise HTTPException(status_code=401, detail='Invalid Token')
 
-    
     def auth_wrapper(self, auth: HTTPAuthorizationCredentials = Security(security)):
-        return self.decode_token(auth.credentials)        
+        return self.decode_token(auth.credentials)
+
 
 auth_handler = AuthHandler()

--- a/backend/app/server/controllers/user.py
+++ b/backend/app/server/controllers/user.py
@@ -1,15 +1,15 @@
 from bson.objectid import ObjectId
 from ..database import users_collection
-from passlib.context import CryptContext
-from fastapi import HTTPException,status,Depends
+from fastapi import HTTPException, status
 from .auth import auth_handler
 
 # helpers
 
+
 def user_helper(user) -> dict:
     return {
         "user_id": str(user["_id"]),
-        "name" : user["name"],
+        "name": user["name"],
         "email": user["email"],
         "profile_picture": user["profile_picture"],
         "description": user["description"]
@@ -26,7 +26,7 @@ async def retrieve_users():
 
 # Add a new user into to the database
 async def add_user(user_data: dict) -> dict:
-    if await users_collection.find_one({"email":user_data['email']}):
+    if await users_collection.find_one({"email": user_data['email']}):
         raise HTTPException(status_code=404, detail='Account already exists with the email')
     hashed_password = auth_handler.get_password_hash(user_data["password"])
     user_data["password"] = hashed_password
@@ -34,13 +34,15 @@ async def add_user(user_data: dict) -> dict:
     new_user = await users_collection.find_one({"_id": user.inserted_id})
     return user_helper(new_user)
 
+
 # Login Functionality
 async def login(user_data: dict) -> dict:
-    getUser = await users_collection.find_one({"email":user_data["email"]})
+    getUser = await users_collection.find_one({"email": user_data["email"]})
     if (getUser is None) or (not auth_handler.verify_password(user_data["password"], getUser["password"])):
         raise HTTPException(status_code=status.HTTP_401_UNAUTHORIZED, detail='Invalid Credentials')
     token = auth_handler.create_access_token(str(getUser["email"]))
     return {"access_token": token, "token_type": "bearer"}
+
 
 # Retrieve a user with a matching ID
 async def retrieve_user(id: str) -> dict:
@@ -54,7 +56,7 @@ async def update_user(email: str, data: dict):
     # Return false if an empty request body is sent.
     if len(data) < 1:
         return False
-    user = await users_collection.find_one({"email":email})
+    user = await users_collection.find_one({"email": email})
     if user:
         updated_user = await users_collection.update_one(
             {"email": email}, {"$set": data}

--- a/backend/app/server/models/post.py
+++ b/backend/app/server/models/post.py
@@ -26,7 +26,6 @@ class UpdatePostModel(BaseModel):
     image: Optional[str]
     description: Optional[str]
 
-
     class Config:
         schema_extra = {
             "example": {

--- a/backend/app/server/models/user.py
+++ b/backend/app/server/models/user.py
@@ -1,6 +1,7 @@
 from typing import Optional
 from pydantic import BaseModel, EmailStr, Field
 
+
 class UserSchema(BaseModel):
     name: str = Field(...)
     email: EmailStr = Field(...)
@@ -11,13 +12,14 @@ class UserSchema(BaseModel):
     class Config:
         schema_extra = {
             "example": {
-                "name" : "John Doe",
+                "name": "John Doe",
                 "email": "jdoe@x.edu.ng",
                 "password": "password",
                 "profile_picture": "url_of_image",
                 "description": "This is John Doe"
             }
         }
+
 
 class LoginSchema(BaseModel):
     email: EmailStr = Field(...)
@@ -26,21 +28,22 @@ class LoginSchema(BaseModel):
     class Config:
         schema_extra = {
             "example": {
-                "email" : "jdoe@s.edu.org",
-                "password" : "Jdoe@123"
+                "email": "jdoe@s.edu.org",
+                "password": "Jdoe@123"
             }
         }
+
+
 class UpdateUserModel(BaseModel):
     name: Optional[str]
     password: Optional[str]
     profile_picture: Optional[str]
     description: Optional[str]
 
-
     class Config:
         schema_extra = {
             "example": {
-                "name" : "John Doe",
+                "name": "John Doe",
                 "password": "password",
                 "profile_picture": "url_of_image",
                 "description": "This is John Doe"

--- a/backend/app/server/routes/post.py
+++ b/backend/app/server/routes/post.py
@@ -17,6 +17,7 @@ from ..models.post import (
 
 router = APIRouter()
 
+
 @router.post("/create", response_description="Post added into the database")
 async def add_post_data(post: PostSchema = Body(...), current_user=Depends(auth_handler.auth_wrapper)):
     post = jsonable_encoder(post)

--- a/backend/app/server/routes/user.py
+++ b/backend/app/server/routes/user.py
@@ -1,12 +1,12 @@
 from fastapi import APIRouter, Body, status, Depends
 from fastapi.encoders import jsonable_encoder
-from fastapi.security import OAuth2PasswordBearer,OAuth2PasswordRequestForm
+from fastapi.security import OAuth2PasswordBearer
 from ..controllers.user import (
     add_user,
     login,
     delete_user,
     retrieve_user,
-    update_user,    
+    update_user
 )
 
 from ..controllers.auth import auth_handler
@@ -19,17 +19,20 @@ from ..models.user import (
 router = APIRouter()
 oauth2_scheme = OAuth2PasswordBearer(tokenUrl="login")
 
+
 @router.post("/register", response_description="User data added into the database", status_code=status.HTTP_201_CREATED)
 async def add_user_data(user: UserSchema = Body(...)):
     user = jsonable_encoder(user)
     new_user = await add_user(user)
     return ResponseModel(new_user, "User added successfully.")
 
+
 @router.post("/login", response_description="User Logged in Successfully", status_code=status.HTTP_201_CREATED)
 async def user_login(user_details: LoginSchema = Body(...)):
     user_details = jsonable_encoder(user_details)
     login_response = await login(user_details)
-    return ResponseModel(login_response,"User Logged in Successfully")
+    return ResponseModel(login_response, "User Logged in Successfully")
+
 
 @router.delete("/delete", response_description="Delete user from the database")
 async def delete_user_data(current_user=Depends(auth_handler.auth_wrapper)):
@@ -42,6 +45,7 @@ async def update_user_data(updated_user: UpdateUserModel = Body(...), current_us
     updated_user = jsonable_encoder(updated_user)
     new_user = await update_user(current_user, updated_user)
     return ResponseModel(new_user, "User updated successfully.")
+
 
 @router.get("/details", response_description="Get user details from the database")
 async def details_user_data(user_id: str):

